lib/block.ml

```diff
@@ -324,6 +324,9 @@ let guess_ocaml_kind b =
 let toplevel ~file ~line lines = Toplevel (Toplevel.of_lines ~line ~file lines)
 
 let eval t =
+  let re_tab = Re.(compile (char '\t')) in
+  let contents = List.map Re.(replace_string re_tab ~by:"      ") t.contents in
```

Comment:
Could you add a comment saying why ?

Reply:
I added a warning, it sounds enough to me (can be verbose but we don't have a non-duplicate mechanism yet)